iFixit/react-commerce
# Use Storefront delegate token for server side requests

closes #1017 

This PR enables the use of a Shopify storefront delegate token for server side requests

## QA

1. Open Vercel preview
2. Verify that you can see product pages (if the token doesn't have the right permissions you'd get 500s)
3. Open production strapi instance
4. Verify that US store (`Store > United States`) delegate access token field is populated

    <img width="832" alt="image" src="https://user-images.githubusercontent.com/4640135/203058213-d5988678-c279-497b-b7d3-220996948d58.png">

🚨 **Please note that the delegate token should be considered a secret, and thus should not be shared, nor posted on Github.**
